### DISCUSSION TITLE
Sample Organizations

### DISCUSSION THREAD

**Discussion by edgarithm** [eoscostarica/gGoods#266] [Ideas]
Lets include "causes"  as well as organizations in the sample data. 

For example:
 - #standuptocancer
 - AIDS Walk
 - Hurricane Relief

**Comment by ldrojas** [comment]
I think we should skip any direct reference to existing orgs. We can use mock data for this. For instance: "Rainforest Initiative" instead of "Animal Rescue Center CR" or "AIDS Program" instead of "AIDS Walk"

**Reply by ldrojas** [reply]
My proposed mock orgs:

1) Rainforest Initiative 
2) AIDS Program
3) Dog Shelter
4) Hurricane Relief

**Comment by xavier506** [comment]
we need the following info for an org

- Org Name
- website
- Address
- Phone 
- description 
- photo
- category

**Reply by xavier506** [reply]
optional fields : 

- impact
- history 
- location

**Reply by ldrojas** [reply]
Definitive info:

- name
- impact
- description (objectives and history)
- website
- location
- email
